examples/workers/main.go

```diff
@@ -106,8 +111,13 @@ func main() {
 	// stopping workers
 	workers.Stop()
 
+	ctx, cancel = context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
 	// wait until all workers will be stopped
-	workers.Wait()
+	if err = workers.Wait(ctx); err != nil {
+		fmt.Printf("[%s] error of completion\n", time.Now().Format("15:04:05"))
```

Comment:
srsly?

Reply:
This is examples. Looks normal for examples.